cloud.amazon/build.xml

```diff
@@ -25,7 +25,6 @@
 
     <property name="resourcesdir" value="./src/org/netbeans/modules/cloud/amazon/resources/"/>
     <target name="build-init" depends="projectized.build-init">
-        <unzip src="external/cloud.amazon-external-resources.zip" dest="${resourcesdir}"/>
     </target>
```

Comment:
Probably, we should remove the following? they are unused, right?
```
    <property name="resourcesdir" value="./src/org/netbeans/modules/cloud/amazon/resources/"/>
    <target name="build-init" depends="projectized.build-init">
    </target>
```

Reply:
Had removed now those extra lines